src/spec/page-objects/connection-settings-page.js

```diff
@@ -0,0 +1,21 @@
+class ConnectionSettingsPage {
+  constructor() {
+    this._url = '/#/settings/connection-settings';
+    this._apiKeyContainer = $('.apiKeyContainer p');
+    this._copyApiKeyButton = $('.apiKeyContainer button');
+  }
+
+  getApiKey() {
+    return this._apiKeyContainer.getText();
+  }
+
+  clickCopyApiKey() {
+    return this._copyApiKeyButton.click();
+  }
+
+  getApiKeyToCopy() {
+    var apiKeyToCopy = $('.copyContainer .inputContainer input');
```

Comment:
I thought it wouldn't work because that html portion is loaded when the modal is shown but I just tested it and it worked. I'll change it